_includes/common-scripts.html

```diff
@@ -0,0 +1,11 @@
+<script src="{{baseurl}}/assets/vendor/jquery-3.3.1.min.js"></script>
+<script src="{{baseurl}}/assets/vendor/bootstrap-4.1.3-dist/js/bootstrap.bundle.js"></script>
+
+<script>
+    jQuery(document).ready(function () {
```

Comment:
Try to be consistent, don't include spaces inside `function()`, as you do right after this

Reply:
👍